index.html

```diff
@@ -0,0 +1,15 @@
+<html>
```

Comment:
This is missing `<!DOCTYPE HTML>` which is required on HTML pages

Reply:
ack! https://github.com/flutter/flutter.github.io/commit/1039f7a79dacbebabdceef0342222dfcfbb2bbe9